README.md

```diff
@@ -409,8 +440,9 @@ router.find('GET', '/example', { host: 'fastify.io', version: '1.x' })
 ```
 
 <a name="pretty-print"></a>
-#### prettyPrint([{ commonPrefix: false }])
+#### prettyPrint([{ commonPrefix: false, includeMeta: true || [] }])
```

Comment:
The `includeMeta` should be removed if I have understood

Reply:
The `includeMeta` option is what triggers the display, otherwise just the plain tree of routes is returned. It also allows for an array of keys to include (e.g. if you're just interested in seeing `onRequest` hooks, you can pass `{includeMeta:['onRequest']}` to only show that property if it is set on a route.

Reply:
I have understood. Is not it a shortcut to skip the `buildPrettyMeta` function?